x-pack/plugins/security_solution/public/management/components/console/components/command_usage.tsx

```diff
@@ -27,42 +19,61 @@ export const CommandInputUsage = memo<Pick<CommandUsageProps, 'commandDef'>>(({
     return getArgumentsForCommand(commandDef).map((usage) => {
       return (
         <EuiText size="s">
-          <EuiBadge>{commandDef.name}</EuiBadge>
-          <ConsoleCodeBlock>{usage}</ConsoleCodeBlock>
+          <ConsoleCodeBlock>{`${commandDef.name} ${usage}`}</ConsoleCodeBlock>
         </EuiText>
       );
     });
   }, [commandDef]);
 
+  const additionalProps = useMemo(
```

Comment:
I would say there is no need for `useMemo` here since this is static and will not change on renders.

Reply:
done in this commit: b5d7ca467565f1c6bcd65211b2f137a3593d46e1